dal/src/test/java/com/craftsmanship/tfm/dal/model/ItemEntityTest.java

```diff
@@ -45,13 +45,13 @@ public void tearDown() throws Exception {
 	@Test
 	public void whenSaveItem_thenReturnItem() {
 		Item savedItem = this.entityManager.persistAndFlush(item);
-		assertThat(savedItem.getDescription()).isEqualTo("Item description");
+		assertThat(savedItem.getName()).isEqualTo("Item description");
```

Comment:
If you are comparing names... why not the rest of attributes?

Reply:
Done